docker/service/src/main/java/uk/nhs/adaptors/scr/config/SpineConfiguration.java

```diff
@@ -13,4 +13,5 @@
 public class SpineConfiguration {
     private String url;
     private String sampleEndpoint;
+    private String acsEndpoint;
```

Comment:
Probably we don't need a separate acsEndpoint. We are going to have just one Spine URL/Endpoint that we're going to use.